corehq/apps/app_manager/templates/app_manager/partials/case_list.html

```diff
@@ -30,14 +30,18 @@
         <option value="no">Don't Use Case Tiles</option>
         <option value="yes">Use Case Tiles</option>
     </select>
-    <label class="checkbox" data-bind="visible: useCaseTiles() == 'yes'">
+    {% endif %}
+    {% if request|toggle_enabled:'CASE_LIST_TILE' or request|toggle_enabled:'CASE_LIST_CUSTOM_XML'%}
+    <div data-bind="visible: useCaseTiles() == 'yes' || window.toggles.CASE_LIST_CUSTOM_XML">
```

Comment:
Yeah, I was struggling to figure out the cleanest way to do this logic. The `CASE_LIST_CUSTOM_XML` check is needed here in the event that the user has both `CASE_LIST_CUSTOM_XML` and `CASE_LIST_TILE` enabled. If `CASE_LIST_TILE` is enabled, then it might be the case that `useCaseTiles() == 'yes'` is `false`, but I still want the checkbox to be visible if `CASE_LIST_CUSTOM_XML` is enabled.

Ah, I just realized this is probably the way to go:
```html
<div data-bind="visible: window.toggles.CASE_LIST_CUSTOM_XML || useCaseTiles() == 'yes' ">
```